src/Symfony/Component/Translation/Translator.php

```diff
@@ -198,6 +198,10 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
      */
     public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null)
     {
+        $parameters = array_merge(array(
```

Comment:
@ogizanagi See this comment: https://github.com/symfony/symfony/issues/7029#issuecomment-13332231